pkg/init/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-default: push
+include ../package.mk
 
-ORG?=linuxkit
 IMAGE=init
-DEPS=Dockerfile init $(wildcard etc/*) $(wildcard etc/init.d/*) usermode-helper.c
-
-HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
-
-tag: $(DEPS)
-	docker build --no-cache --network=none -t $(ORG)/$(IMAGE):$(HASH) .
-
-push: tag
-	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
-	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+DEPS=init $(wildcard etc/*) $(wildcard etc/init.d/*) usermode-helper.c
```

Comment:
super minor nit, but I think it's nicer if the individual files are listed before wildcards and not mixed like this (again, I realises this was copied from the original, but no reason to not improve it)